source/install/mattermost-omnibus.rst

```diff
@@ -0,0 +1,128 @@
+Installing Mattermost Omnibus
+============================
+
+Overview
+---------
+
+Mattermost Omnibus is a Debian package that packages the individual components of a Mattermost deployment into a single installation process. The package leverages the ``apt`` package manager to install and keep the components of the platform updated, using a custom CLI and ansible recipes to link those components together and configure them.
+
+Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions.
+
+Prerequisites
+-------------
+
+- A clean Ubuntu server (18.04. Or 20.04)
+- A domain name pointing to that server
+- Root level access to the server
+
+Installing Mattermost Omnibus
+------------------------------
+
+The Omnibus repositories are configured using a cURL command:
+
+``curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash``
+
+This command configures the repositories needed for a ``PostgreSQL`` database, an ``nginx`` web server to act as a proxy, and ``certbot`` to issue and renew the SSL certificate.
+
+In order to issue that certificate, the installer requests a domain name and an email address from us. These are used to generate the certificate and deliver any related communications respectively.
+
+Installing Mattermost Omnibus
+------------------------------
+
+If you're installing Omnibus, and have a domain and SSL use:
+
+``sudo apt install mattermost-omnibus``.
+
+After all the packages are installed, Omnibus runs the ansible scripts that configure all the platform components and starts the server. To access Mattermost, open a browser, navigate to your domain, and create the System Admin user to start using the platform.
+
+Updating Mattermost Omnibus
+-----------------------------
+
+Mattermost Omnibus is integrated with the ``apt`` package manager. When a new Mattermost version is released, run:
+
+``sudo apt update && sudo apt upgrade``
+
+This downloads the latest version of Mattermost and updates your Mattermost platform.
+
+Configuring Mattermost Omnibus
+----------------------------------------
+
+.. note::
+  Plugin uploads and HTTPS are enabled by default. These settings are modified in the ``yaml`` file using `mmctl config edit <https://docs.mattermost.com/administration/mmctl-cli-tool.html#mmctl-config-edit>`__.
```

Comment:
Adding another property configured by default and changing config method as the `yaml` file needs to be modified directly:

```suggestion
  Plugin uploads, local mode and HTTPS are enabled by default. These settings are modified in the ``yaml`` file as described below.
```